content/influxdb/cloud-dedicated/admin/clusters/get.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Get cluster information
+description: >
+  Use the
+  [`influxctl cluster get <CLUSTER_ID>` command](/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/get/) to view information about your InfluxDB Cloud Dedicated cluster.
+menu:
+  influxdb_cloud_dedicated:
+    parent: Manage clusters
+weight: 202
+list_code_example: |
+  ```sh
+  influxctl cluster get 0x000000-xy00-0xy-x00-0x00y0000000
+  ```
+---
+
+Use the [`influxctl cluster get` CLI command](/influxdb/cloud-dedicated/reference/cli/influxctl/get/)
+to view information about your {{< product-name omit=" Clustered" >}} cluster.
+
+1.  If you haven't already, [download and install the `influxctl` CLI](/influxdb/cloud-dedicated/reference/cli/influxctl/#download-and-install-influxctl), and then [configure a connection profile](/influxdb/cloud-dedicated/reference/cli/influxctl/#configure-connection-profiles) for your cluster.
+2.  Run `influxctl cluster get` with the following:
+
+    - Cluster ID
+    - _Optional_: [Output format](#output-formats)
+
+    ```sh
+    influxctl cluster get --format table <CLUSTER_ID>
+    ```
```

Comment:
~~~suggestion
{{% code-placeholders "CLUSTER_ID" %}}
```sh
influxctl cluster get --format table CLUSTER_ID
```
{{% /code-placeholders %}}

Replace {{% code-placeholder-key %}}`CLUSTER_ID` {{% /code-placeholder-key %}} with the
ID of the cluster you want to view information about.
~~~